core/src/test/scala/org/apache/spark/sql/store/TokenizationTest.scala

```diff
@@ -318,7 +318,7 @@ class TokenizationTest
           assert(r.get(0) == r.get(1) && r.get(2) == i)
         })
       }
-      assert(cacheMap.size() == 1)
+      assert(cacheMap.size() == 0)
```

Comment:
Pretty sure this test passed when the changes were made. Not sure what happened here but these changes look correct.